lib/fastlane/plugin/wpmreleasetoolkit/helper/ios/ios_l10n_helper.rb

```diff
@@ -0,0 +1,178 @@
+require 'yaml'
+require 'tmpdir'
+
+module Fastlane
+    module Helpers
+      class IosL10nHelper
+        SWIFTGEN_VERSION = '6.4.0'
+        DEFAULT_BASE_LANG = 'en'
+        CONFIG_FILE_NAME = 'swiftgen-stringtypes.yml'
+
+        attr_reader :install_path
+        attr_reader :version
+
+        # @param [String] install_path The path to install SwiftGen to. Usually something like "#{PROJECT_DIR}/vendor/swiftgen/#{SWIFTGEN_VERSION}".
+        #        It's recommended to provide an absolute path here rather than a relative one, to ensure it's not dependant on where the action is run from.
+        # @param [String] version The version of SwiftGen to use. This will be used both:
+        #        - to check if the current version located in `install_path`, if it already exists, is the expected one
+        #        - to know which version to download if there is not one installed in `install_path` yet
+        #
+        def initialize(install_path:, version: SWIFTGEN_VERSION)
+          @install_path = install_path
+          @version = version || SWIFTGEN_VERSION
+        end
+
+        # Check if SwiftGen is installed in the provided `install_path` and if so if the installed version matches the expected `version`
+        #
+        def check_swiftgen_installed
+          return false unless File.exists?(swiftgen_bin)
+          vers_string = `#{swiftgen_bin} --version`
+          # SwiftGen v6.4.0 (Stencil v0.13.1, StencilSwiftKit v2.7.2, SwiftGenKit v6.4.0)
```

Comment:
```suggestion
          # The SwiftGen version string has this format:
          #
          # SwiftGen v6.4.0 (Stencil v0.13.1, StencilSwiftKit v2.7.2, SwiftGenKit v6.4.0)
```

Suggesting this because it took me a few extra seconds to understand the comment.